scripts/package.js

```diff
@@ -0,0 +1,18 @@
+const process = require('process');
+const path = require('path');
+const baseDir = path.join(__dirname, '..');
```

Comment:
Separate imports from rest.

```suggestion

const baseDir = path.join(__dirname, '..');
```